engine/resources/packagekit/packagekit.go

```diff
@@ -972,5 +972,5 @@ func IsMyArch(arch string) (bool, error) {
 	if goarch == archUtil.Any { // special value that corresponds to noarch
```

Comment:
I wonder if this should be catching this case 🤔 
This code could be optimised a little bit I guess:
```suggestion
	return goarch == runtime.GOARCH || goarch == "ANY" || goarch == archUtil.Any, nil
```